src/molecules/gv-cron-editor.js

```diff
@@ -0,0 +1,615 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import '../atoms/gv-date-picker';
+import '../organisms/gv-tabs';
+import '../atoms/gv-input';
+import '../atoms/gv-tag';
+import { css, LitElement, html } from 'lit-element';
+import { skeleton } from '../styles/skeleton';
+import { InputElement } from '../mixins/input-element';
+import { set } from 'object-path';
+import { repeat } from 'lit-html/directives/repeat';
+import { withResizeObserver } from '../mixins/with-resize-observer';
+import { isValidCron } from '../lib/cron-expression';
+import { dispatchCustomEvent } from '../lib/events';
+
+/**
+ * Cron editor component
+ *
+ * ## Details
+ *
+ * @fires gv-cron-editor:input - input events with the `value` on `detail`
+ *
+ * @attr {String} label - cron editor label
+ * @attr {String} value - cron expression
+ * @attr {String} mode - the opened mode (if have value, the export mode will be open)
+ *
+ * @attr {Boolean} [clipboard=false]- true if field has clipboard button
+ * @attr {Boolean} [autofocus=false] - true to put the focus on the input
+ * @attr {Boolean} [readonly=false] - true if field is readonly mode
+ */
+export class GvCronEditor extends withResizeObserver(InputElement(LitElement)) {
+
+  static get properties () {
+    return {
+      ...super.properties,
+      mode: { type: String },
+      _value: { type: String, attribute: false },
+      _currentDocumentationId: { type: String, attribute: false },
+      _small: { type: Boolean },
+      _truncateTabs: { type: Boolean },
+    };
+  }
+
+  constructor () {
+    super();
+    this._value = '';
+    this.readonly = false;
+    this.autofocus = false;
+    this.clipboard = false;
+    this._modes = ['seconds', 'minutes', 'hourly', 'daily', 'weekly', 'monthly', 'yearly', 'pro'];
+    this._timeRange = [...Array(60).keys()];
+    this._days = [
+      { value: 'MON', label: 'Monday' },
```

Comment:
do you think these label should be translated ?

Reply:
@aelamrani @phiz71 What do you think ?

Reply:
not useful, it is an admin component